plugins/kotlin-as-java/src/test/kotlin/KotlinAsJavaPluginTest.kt

```diff
@@ -88,9 +92,9 @@ class KotlinAsJavaPluginTest : BaseAbstractTest() {
                     .map { it.content }
 
                 val children = contentList.flatMap { content ->
-                    content.mainContents.single { it is ContentGroup }.children
-                        .filterIsInstance<ContentTable>()
-                        .filter { it.children.isNotEmpty() }
+                    (content.findTabWithType(BasicToggleableContentType.FUNCTION)
+                        ?: throw IllegalStateException("A function tab is not found")).children[0].children.filterIsInstance<ContentTable>()
+                        .flatMap { it.children }
```

Comment:
It's a bit difficult to understand where content begins and ends. Let's break it down?
```suggestion
                    val functionTab = content.findTabWithType(BasicToggleableContentType.FUNCTION)
                        ?: throw IllegalStateException("A function tab is not found")
                        
                    functionTab.children[0].children.filterIsInstance<ContentTable>()
                        .flatMap { it.children }
```